maritalk/resources/local.py

```diff
@@ -231,11 +231,17 @@ def generate(
 
         if response.ok:
             return response.json()
+
+        if response.headers.get("content-type") == "application/json":
+            data = response.json()
+            if "error" in data:
```

Comment:
I'm not sure if I understood.. if there is an error key in the data dictionary, return data?

Reply:
Exactly! This means that the error message has the shape of our application's errors. But I don't see any problem of removing it.

Reply:
maybe just add a comment explaining what these lines do?